packages/workbox-expiration/Plugin.mjs

```diff
@@ -131,6 +132,13 @@ class Plugin {
     const cacheExpiration = this._getCacheExpiration(cacheName);
     cacheExpiration.expireEntries();
 
+    // Update the metadata for the request URL to the current timestamp,
+    // but don't `await` it as we don't want to block the response.
+    const updateTimestampDone = cacheExpiration.updateTimestamp(request.url);
+    if (event) {
+      event.waitUntil(updateTimestampDone);
```

Comment:
You're going to need to put this in a `try`/`catch`, à la https://github.com/GoogleChrome/workbox/pull/1457/files

[Samsung Internet 6.4](https://developer.samsung.com/internet/android/releases), which I believe we're still trying to support, breaks when you call `event.waitUntil()` asynchronously: https://github.com/GoogleChrome/workbox/issues/1435#issuecomment-385690454